pipes/WDL/tasks/tasks_nextstrain.wdl

```diff
@@ -37,21 +39,22 @@ task nextclade_one_sample {
         fi
 
         nextclade run \
-            --input-fasta "~{genome_fasta}" \
-            --input-root-seq ~{default="reference.fasta" root_sequence} \
-            --input-tree ~{default="tree.json" auspice_reference_tree_json} \
-            --input-qc-config ~{default="qc.json" qc_config_json} \
-            --input-gene-map ~{default="genemap.gff" gene_annotations_json} \
-            --input-pcr-primers ~{default="primers.csv" pcr_primers_csv} \
-            --input-virus-properties ~{default="virus_properties.json" virus_properties}  \
+            $DATASET_ARG \
+            ~{"--input-root-seq " + root_sequence} \
+            ~{"--input-tree " + auspice_reference_tree_json} \
+            ~{"--input-qc-config " + qc_config_json} \
+            ~{"--input-gene-map " + gene_annotations_json} \
+            ~{"--input-pcr-primers " + pcr_primers_csv} \
+            ~{"--input-virus-properties " + virus_properties}  \
+            --output-basename "~{basename}" \
             --output-json "~{basename}".nextclade.json \
             --output-tsv  "~{basename}".nextclade.tsv \
-            --output-tree "~{basename}".nextclade.auspice.json
-        cp "~{basename}".nextclade.tsv input.tsv
+            --output-tree "~{basename}".nextclade.auspice.json \
+            "~{genome_fasta}"
```

Comment:
@dpark01 looks like you may need to add an `--output-all` (`-O`) flag to define the output dir--now a [required arg in v2](https://github.com/nextstrain/nextclade/blob/master/CHANGELOG.md#nextclade-cli)